GoogleDataTransport/GDTCCTLibrary/GDTCCTNanopbHelpers.m

```diff
@@ -138,7 +138,7 @@ gdt_cct_LogEvent GDTCCTConstructLogEvent(GDTCOREvent *event) {
   gdt_cct_LogEvent logEvent = gdt_cct_LogEvent_init_default;
   logEvent.event_time_ms = event.clockSnapshot.timeMillis;
   logEvent.has_event_time_ms = 1;
-  logEvent.event_uptime_ms = event.clockSnapshot.uptime;
+  logEvent.event_uptime_ms = [event.clockSnapshot uptimeMilliseconds];
```

Comment:
Is there a way to ascertain if these are milliseconds and not nanoseconds or seconds?

Reply:
It is asserted [here](https://github.com/firebase/firebase-ios-sdk/blob/505093c87201a1b2314fe9cc193b6ae93ed6bbed/GoogleDataTransport/GDTCCTTests/Unit/GDTCCTNanopbHelpersTest.m#L136).